package.json

```diff
@@ -93,7 +93,7 @@
   ],
   "engines": {
     "node": ">=10.16.3 <10.17.0",
-    "yarn": ">=1.17.3 <1.18.0"
+    "yarn": ">=1.17.3 <1.18.0 || 1.19.1"
```

Comment:
Let's break that up into a different PR.

Reply:
If you upgrade `yarn` here, I would suggest also upgrading it in the `volta` section.